flink-streaming-connectors/flink-connector-elasticsearch2/pom.xml

```diff
@@ -55,11 +55,11 @@ under the License.
             <version>${elasticsearch.version}</version>
         </dependency>
 
-		<dependency>
-			<groupId>com.fasterxml.jackson.core</groupId>
-			<artifactId>jackson-core</artifactId>
-			<version>2.7.2</version>
-		</dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-core</artifactId>
+            <version>2.7.4</version>
```

Comment:
Replace with ${jackson.version} if the parent pom value for this is being changed to 2.7.2